source/dsl/import_parsers/ImportParser.py

```diff
@@ -59,7 +59,7 @@ def _parse_dataset(key: str, dataset_specs: dict, symbol_table: SymbolTable, res
             symbol_table.add(key, SymbolType.DATASET, dataset)
         except KeyError as key_error:
             raise KeyError(f"{key_error}\n\nAn error occurred during parsing of dataset {key}. "
-                           f"Parameter {key_error.args[0]} was not defined in definitions:datasets:{key}:params.")
+                           f"Parameter {key_error.args[0]} was not defined in definitions/datasets/{key}/params.")
```

Comment:
with `/` sign it looks to me a bit like path, but if you think this is better, that's also fine :)

Reply:
since we use the / sign in other error messages as well, I thought this was more consistent?

Reply:
that's a good point. maybe we should have it all the other way? this is a minor comment anyway, just thinking about what would be most user-friendly. we could go for this now and change it later if someone else finds it confusing? 🙏 😊

Reply:
Actually, even though this is small, this is something where it is nice to poll other user's opinions. I'll try to do this on Slack